os/dutils.sh

```diff
@@ -287,7 +313,8 @@ function TRAP_RUN()
 	echo "Executing: $TRAPCMD" | tee $OUTPUTFILE
 	echo "" >> $OUTPUTFILE
 	# execute TRAP script
-	docker run --rm ${DOCKER_OPT} -v ${TOPDIR}:/root/tizenrt -it -w /root/tizenrt/tools/trap --privileged tizenrt/tizenrt:1.5.6 python3.7 $TRAPCMD | tee -a $OUTPUTFILE
+	CHECK_DOCKER_IMAGE 1.5.6
+	docker run --rm ${DOCKER_OPT} -v ${TOPDIR}:/root/tizenrt -it -w /root/tizenrt/tools/trap --privileged ${DOCKER_IMAGE}:1.5.6 python3.7 $TRAPCMD | tee -a $OUTPUTFILE
```

Comment:
`${DOCKER_IMAGE}:1.5.6`
Why do you use hard-coded version instead of DOCKER_VERSION?

Reply:
I need to check why it is only 1.5.6
I'll check version.

Reply:
I think you should check this.
https://github.com/Samsung/TizenRT/pull/5819

Reply:
Thank you, It applied in #5819